src/test/regress/spec/isolation_update_node.spec

```diff
@@ -8,6 +8,9 @@ setup
 
 teardown
 {
+    UPDATE pg_dist_node SET metadatasynced = 't' WHERE nodeport = 57637;
```

Comment:
Well, as you noted, given that we have to use nodes that do not exists in citus_update_node, we really need something like this.

But, doing this at the teardown seems hard to understand where we actually need to do this. Can we maybe have one explicit step like: `manually-fix-metadata` and (a) use it only when necessary (b) make it explicit what we are doing